src/lv_core/lv_obj.c

```diff
@@ -55,6 +55,20 @@ static const void * event_act_data;           /*Stores the data passed to the ev
 /**********************
  *      MACROS
  **********************/
+#if USE_LV_EXTENDED_CLICK_AREA
+/**
+ * Update coordinates of extended clickable area from object's coordinates and ext_paddings
+ * @param coords coordinates of an object
+ * @param ext_coords extended coordinates, which will be updated
+ * @param paddings paddings of extended clickable area
+ */
+#define UPDATE_EXT_COORDS(coords, ext_coords, paddings) do{\
```

Comment:
This should be made into a `static` or `static inline` function.

Reply:
ok, i'll change to static inline